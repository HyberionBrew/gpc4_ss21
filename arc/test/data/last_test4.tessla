in a: Events[Unit]
in x: Events[Unit]
in z: Events[Int]
def y := last(z,a)
def o := last(y,x)
out o
