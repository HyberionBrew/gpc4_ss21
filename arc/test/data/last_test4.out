21: o = 33
